src/Stripe.net/Entities/Events/EventData.cs

```diff
@@ -12,13 +15,19 @@ public class EventData : StripeEntity<EventData>
         /// </summary>
         [JsonProperty("object")]
         [JsonConverter(typeof(StripeObjectConverter))]
+#if NET6_0_OR_GREATER
+        [STJS.JsonPropertyName("object")]
```

Comment:
I think I'll learn the answer soon enough but won't the lack of StripeObjectConverter make this model not directly deserializeable?

Reply:
Found it, the converter is on IHasObject

Reply:
Yea; STJ doesn't have the ItemConverterType parameter in JsonPropertyName for converting objects inside collections; the solution for that is to add the converter to the inner type itself.  Because of this, the wholesome tests are written to validate interfaces have converters (i.e. for polymorphic resources in lists in other resources); this one could have gone either way but I wanted to keep it consistent with the other interfaces.